examples/snippets/modules/ROOT/examples/akkastreams-scala/build.sbt

```diff
@@ -0,0 +1,42 @@
+//tag::docs-projectSetup-example[]
+import sbt._
+import sbt.Keys._
+
+lazy val sensorData =  (project in file("."))
+    .enablePlugins(CloudflowAkkaStreamsApplicationPlugin)
+    .settings(
+//end::docs-projectSetup-example[]
+      libraryDependencies ++= Seq(
+        "com.lightbend.akka"     %% "akka-stream-alpakka-file"  % "1.1.2",
+        "com.typesafe.akka"      %% "akka-http-spray-json"      % "10.1.10",
+        "ch.qos.logback"         %  "logback-classic"           % "1.2.3",
+        "com.typesafe.akka"      %% "akka-http-testkit"         % "10.1.10" % "test",
```

Comment:
Have you checked, for instance by running these examples, that it does not start with many versions of Akka libraries on classpath? We depend on `val AkkaHttp     = "10.1.9"` in cloudflow itself. Probably best to keep that the same.